### DISCUSSION TITLE
Map marker for tracking range

### DISCUSSION THREAD

**Discussion by TheAngryRogue** [smasherprog/EqTool#39] [Ideas]
Curious if it's possible to add an option to select my tracking level, have it combine with my class and put a simple line circle around the center of the characters position on the map giving an estimated area of effect. Would help with planning sweeping routes for farming non stationary targets.

Add the option to draw simple lines on the map as a reminder of what path to follow would be great. 

If not possible or considered against the ToS of P99 then I apologize and will remove thread.

**Comment by smasherprog** [comment]
I was planning on doing this EXACT thing! Its on my to do list and will get done eventually

**Comment by smasherprog** [comment]
I was doing a few Peggy cloak farms and it would have been very helpfull to know what ground i was covering.

**Comment by TheAngryRogue** [comment]
Loving the tracking option. It is exactly what I was looking for. Thank you!
